pkg/handler/handler.go

```diff
@@ -205,6 +207,31 @@ func (handler *Handler) Update(obj client.Object) error {
 	return nil
 }
 
+func (handler *Handler) GetBootstrapUser(instance client.Object) (*corev1.Secret, error) {
+	bootstrapUser := &corev1.Secret{}
+	names := appsv1beta4.Names{Object: instance}
+	if err := handler.Client.Get(context.TODO(), types.NamespacedName{Name: names.BootstrapUser(), Namespace: instance.GetNamespace()}, bootstrapUser); err != nil {
+		return nil, err
+	}
+	return bootstrapUser, nil
+
+}
+
+func GetUsernameAndPasswordFromBootstrapUser(bootstrapUser *corev1.Secret) (string, string, error) {
```

Comment:
I want `handler.go` to handle only create and update tasks, so this function should move other where

Reply:
done